### ISSUE TITLE
Warning "Container#eachAtRule is deprecated" after installing v0.19.0

### ISSUE THREAD

**Issue by greenthoms** [sintaxi/harp#497]
Installed Harp for the first time and encountered the following warning:

```
Container#eachAtRule is deprecated. Use Container#walkAtRules instead.
Container#eachRule is deprecated. Use Container#walkRules instead.
Container#eachDecl is deprecated. Use Container#walkDecls instead.
```

**Comment by kennethormandy** [comment]
It will be fixed in the next release, it’s coming from Autoprefixer. Unfortunately updating it and taking the fix would also mean removing Node v0.10.x support, which we weren’t ready to do at the time. There’s nothing going wrong, though. :) Thanks for opening an issue!

**Comment by tplusrex** [comment]
nice!
